src/server/schemas/wallet/index.ts

```diff
@@ -27,6 +27,12 @@ export const walletWithAAHeaderSchema = Type.Object({
     description:
       "Smart account factory address. If omitted, engine will try to resolve it from the chain.",
   }),
+  "x-account-salt": Type.Optional(
+    Type.String({
+      description:
+        "Smart account salt in hex format. Useful when creating multiple accounts with the same admin and only needed when deploying the account as part of a userop.",
```

Comment:
Hex format is required unless we hexify non-hex strings. If hex is required, I'd recommend validating it at the API level:
```typescript
Type.RegExp(/^0x[a-fA-F0-9]*$/, {...})
```
instead of
```typescript
Type.String(...)
```

Reply:
let me update the comment actually, it does not need to be in hex format anymore